app/lib/ability.rb

```diff
@@ -5,12 +5,20 @@ def initialize(user)
     return unless user.present?
 
     accessible_organizations = user.accessible_organizations
+    alias_action :response_needed, to: :update
+    alias_action :read, :create, :update, :destroy, to: :crud
 
     if user.is_admin?
       can :manage, :all
     else
       can :manage, [VitaPartner], id: accessible_organizations.pluck(:id)
-      can :manage, [Client, User], vita_partner: accessible_organizations
+      can :manage, User, memberships: { vita_partner: accessible_organizations }
+
+      can :crud, Client, vita_partner: accessible_organizations
+      can :edit_organization, Client do |client|
+        user.can_lead?(client.vita_partner)
+      end
```

Comment:
I believe we can remove this block and just rely on the existing VitaPartner and Client permissions.